MainApp/public/app/views/google-Login.html

```diff
@@ -0,0 +1,104 @@
+<html>
+<head>
+  <meta name="google-signin-client_id" content="713815641075-vge8ouravfnq07fko03patep79djuu0o.apps.googleusercontent.com">
+  <style type="text/css">
+    #logout
+    {
+    float:right;
+    margin-right:20px;
+    height:5px;
+    visibility: hidden;
+    }
+  
+    #my-signin2 
+    {
+         'position':'absolute',
+          'top':'50%',
+          'left':'50%'      
+    }
+  </style>
+
+
+
+
+</head>
+<body>
+  
+  <h1>Welcome to Human Agent Interaction- Machine Learning</h1> 
+  
+  <script type="text/javascript">
+    
+    function logOut(){
+     var auth2 = gapi.auth2.getAuthInstance();
+     auth2.signOut().then(function () {
+     console.log('User signed out.');
+     document.getElementById("logout").style.visibility = "hidden";
+     //document.getElementById("sign-in-success-message").style.visibility = "hidden";
+             
+
+
+    });
+     auth2.disconnect();
+    }  
+
+  </script>
+
+  <a href="#" onclick="logOut();" id="logout">Log Out</a>
+
+  
+  <div id="my-signin2"></div>
+
+  
+  <script>
+    function onSuccess(googleUser) {
+      console.log('Logged in as: ' + googleUser.getBasicProfile().getName());
+      console.log('User Email:'+googleUser.getBasicProfile().getEmail());
+      var email=googleUser.getBasicProfile().getEmail();
+      var request= new XMLHttpRequest();
+      var url="http://localhost:8080/api/adminLogin";
```

Comment:
If you are talking about the hardcoded URL http://localhost:8080/api/adminLogin,  my understanding was that everytime you call http://localhost:8080/login there is a sendFile() function called to send the html file. So changing the URL in the html file on the server would be enough. 
Please correct me if I am wrong. 
Thanks.